packages/alert/__tests__/Alert.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { render, screen } from '@testing-library/react'
+
+import type { AlertProps } from '../src/'
+import Alert from '../src/'
+
+describe('no props', () => {
+  it('renders alert', async () => {
+    render(<Alert className="alert"></Alert>)
+    expect(document.getElementsByClassName('alert')).toHaveLength(1)
```

Comment:
The describe block seems inconsistent with what the test is doing since there is prop `className`